lib/resolver/javascript-universal.js

```diff
@@ -3,6 +3,13 @@ import javascriptFile from './javascript-file.js';
 import liveResolverQuery from './live-resolver-query.js';
 import liveResolverPing from './live-resolver-ping.js';
 
+function getTopModuleName(target) {
+  const isScoped = target.startsWith('@');
```

Comment:
Is it really that simple to differentiate between scoped packages and normal packages? Are there no other edge-cases? At least normal npm packages can't start with an @-sign, but what about others package manger? Maybe I'm thinking it over ...